graph_test.go

```diff
@@ -34,6 +34,19 @@ func TestEmptyWithHTMLLabel(t *testing.T) {
 	}
 }
 
+func TestDeleteNode(t *testing.T) {
```

Comment:
this is just the happy flow, a test is needed for the case the node is not found (also parent situation)

Reply:
The parent situation is already tested below, as well as the child situation ! 
I'm improving my tests, I'll come back later